### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by DarkGhostHunter**
- Added explanation to load from "AppServiceProvider".
- Added Lumen specific notes for installation before **License**.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -174,6 +175,55 @@ Pre-generated example: https://gist.github.com/barryvdh/bb6ffc5d11e0a75dba67
 > Note: You might need to restart PhpStorm and make sure `.phpstorm.meta.php` is indexed.
 > Note: When you receive a FatalException about a class that is not found, check your config (for example, remove S3 as cloud driver when you don't have S3 configured. Remove Redis ServiceProvider when you don't use it).
 
+### Lumen Install
```
Comment (DarkGhostHunter) [inline readme.md]: Crap. This should be a H2. Could someone pull:
`## Lumen Install`
?

File: readme.md [hunk 2]
```diff
@@ -174,6 +175,55 @@ Pre-generated example: https://gist.github.com/barryvdh/bb6ffc5d11e0a75dba67
...
+return [
+    'aliases' => [
+        'CustomAliasOne' => Example\Support\Facades\CustomAliasOne::class,
+        'CustomAliasTwo' => Example\Support\Facades\CustomAliasTwo::class,
+        //...
+    ]
+];
+```
+
+Afyer you run ```php artisan ide-helper:generate```, it's recommended (but not mandatory) to rename `config/app.php` to something else until you have to re-generate the docs or after passing to a production enviroment. Lumen 5.1+ will read this file for configuration parameters if it is present, and may overlap some configurations.
```
Comment (DarkGhostHunter) [inline readme.md]: Typo.  Should say "After".

**Comment by barryvdh** [comment]
You can just push more commits to this branch if you want.

**Comment by DarkGhostHunter** [comment]
Done.

**Comment by barryvdh** [comment]
Thanks!

**Pull request merged**